sycl/test/regression/sycl-ls-unique-device-id-level-zero.cpp

```diff
@@ -0,0 +1,14 @@
+// REQUIRES: level_zero
+
+// RUN: env SYCL_DEVICE_FILTER=level_zero sycl-ls | FileCheck %s --check-prefixes=CHECK-OPENCL
+
+// CHECK-OPENCL-COUNT-1: [level_zero:{{.*}}:0]

```

Comment:
Could you please clarify how this check works?
Also, could you please move tests that require low-level runtime to llvm-test-suite?

Reply:
This // ***-CHECK-# directive only matches the given pattern only # times. In this case, we try to match only once.
I will move these tests to llvm-test-suite.
I wanted to confirm these tests with my PR changeset.

Reply:
So, this check verifies that the tool prints a Level Zero device if Level Zero is available, right?

Reply:
Yes, it verifies that the given pattern (level_zero with device id 0) should be exactly matched once.
For example, if sycl-ls prints `[level_zero:cpu:0]` and `[level_zero:gpu:0]`, this test will fail.